File: sandbox.client/front/ts/model/FileModel.ts
export class FileModel {
  public id: string;
  public name: string;
  public src;

  public static fromFile(f: File): FileModel {
    if (f == null) return null;
    let ret = new FileModel();
    ret.id = null;
    ret.name = f.name;
    return ret;
  }

  public static fromFileModel(f: FileModel): FileModel {
    if (f == null) return null;
    let ret = new FileModel();
    ret.id = f.id;
    ret.name = f.name;
    ret.src = f.src;

    return ret;
  }

}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/FileModel.java
package kz.greetgo.sandbox.controller.model;

public class FileModel {
  public String id;
  public String name;
  public String src;
  public boolean newFile = false;

  public FileModel() {
  }

  public FileModel(String id, String name, String src) {
    this.id = id;
    this.name = name;
    this.src = src;
  }
}
